src/PerformanceTests/Common/Program.cs

```diff
@@ -28,11 +30,18 @@ class Program
 
         static int Main()
         {
+            ServicePointManager.Expect100Continue = false;
+            ServicePointManager.UseNagleAlgorithm = false;
+            ServicePointManager.DefaultConnectionLimit = 150;
+
+            var entryAssembly = Assembly.GetEntryAssembly();
             Thread.CurrentThread.CurrentCulture = CultureInfo.InvariantCulture;
             Thread.CurrentThread.CurrentUICulture = CultureInfo.InvariantCulture;
 
-            XmlConfigurator.Configure(log4net.LogManager.GetRepository(Assembly.GetExecutingAssembly()));
-
+            var logRepository = log4net.LogManager.GetRepository(entryAssembly);
+            var configFileName = Path.GetFileName(new Uri(entryAssembly.CodeBase).LocalPath);
+            XmlConfigurator.Configure(logRepository, new FileInfo($"{configFileName}.config"));
```

Comment:
@danielmarbach do we even need the repository argument?

Reply:
Good question, no idea... It works (TM) ;)